Python/Product/Pylance/source.extension.vsixmanifest

```diff
@@ -9,7 +9,7 @@
     <Icon>PythonProject.ico</Icon>
     <PreviewImage>PythonProjectBig.ico</PreviewImage>
   </Metadata>
-  <Installation SystemComponent="false" AllUsers="true" InstalledByMsi="false">
+  <Installation SystemComponent="true" AllUsers="true" InstalledByMsi="true">
```

Comment:
This is really related to this change but it's kind of in the same spirit. I'm moving the pylance feature branch to be the same as the old stuff until we decide that the extension is going to be outside of VS.